File: 6_random_function/random_function.jl
function first_func()
    return "I am the first function"
end

function second_func()
    return "I am the second function"
end

function third_func()
    return "I am the third function"
end

f = rand((1, 2, 3))

f == 1 ? print(first_func()) : f == 2 ? print(second_func()) : print(third_func())

File: 6_random_function/random_function.cpp
#include <iostream>
#include <stdlib.h>
#include <time.h>

using namespace std;

string first_function() {
    return "I am the first function";
};

string second_function() {
    return "I am the second function";
};

string third_function() {
    return "I am the third function";
};

int main() {
    srand(time(NULL));

    int funcs[3] = {1, 2, 3};
    int rand_index = rand() % 3;
    int rand_func = funcs[rand_index];

    switch (rand_func){
        case 1:
            cout << first_function();
            break;
        case 2:
            cout << second_function();
            break;
        case 3:
            cout << third_function();
            break;
    };

    return 0;
};